frameworks/helloworld/src/main/dist/examples/overlay.yml

```diff
@@ -38,12 +38,14 @@ pods:
         cpus: {{HELLO_CPUS}}
         memory: {{HELLO_MEM}}
         ports:
-          test:
-            port: 4044
+          dummy:
+            port: 1025
+          dynport:
+            port: 0
     tasks:
       server:
         goal: RUNNING
-        cmd: "python3 http_py3responder.py 4044 hello from the overlay network!"
+        cmd: "python3 http_py3responder.py $PORT_DYNPORT hello from the overlay network!"
```

Comment:
This tests that dynamic ports on the overlay network will not collide with user-specified ports. (Dynamic ports start at 1025, so we expect `$PORT_DYNPORT` to be 1026, which is tested by the `get-overlay` task below.